### PR TITLE
My web login page

### PR DESCRIPTION
**Pull request by Michaelazcano**
This is not my official submission, it is a sample.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,7 +8,7 @@
 </head>
 <body>
 
-
+<!-- comment -->
     <style>
```
Comment (clivinghouse) [inline index.html]:
```Suggestion
<p>test</p>
```

**Review by clivinghouse — CHANGES_REQUESTED** [review]
here

File: index.html (hunk 1 above)
Comment (Michaelazcano) [reply index.html]:
```suggestion
<p>test</p>
<style>
```

Comment (Michaelazcano) [reply index.html]: corrected

**Review by Michaelazcano — COMMENTED** [review]
better

**Pull request closed without merging**